docusaurus/docs/e2e-test-a-plugin/get-started.md

```diff
@@ -196,24 +68,15 @@ groupId="package-manager"
 queryString="current-package-manager"
 />
 
-### Step 7: Run tests in CI
+### Step 3: Setup CI
 
-We recommend using a CI workflow to run end-to-end tests to continuously check for breakages. The following Github workflow runs end-to-end tests against a range of Grafana versions for every PR in your Github repository.
+If you choose to add a Github workflow when you scaffolded the plugin, the Playwright end-to-end tests will be run in CI targeting the most recent version of Grafana. To take full advantage of plugin-e2e, it's recommended to follow the instructions in the [CI](./ci.md) guide to run Playwright end-to-end tests targeting all versions of Grafana that your plugin supports.
 
-:::note
-
-This is a generic example based on a backend plugin. You may need to alter or remove some of the steps in the `playwright-tests` job before using it in your plugin.
-
-:::
+## What's next?
 
-<CodeSnippets
-snippets={[
-{ component: ScaffoldPluginE2EDSWorkflowNPM, label: 'npm' },
-{ component: ScaffoldPluginE2EDSWorkflowYarn, label: 'yarn' },
-{ component: ScaffoldPluginE2EDSWorkflowPNPM, label: 'pnpm' }
-]}
-groupId="package-manager"
-queryString="current-package-manager"
-/>
+Next we suggest you checkout the following guides:
 
-![e2e testing of multiple Grafana versions](/img/e2e-version-plugin-dependency.png)
+- [Selecting UI elements in end-to-end tests](./selecting-ui-elements.md)
+- [Configure the resources you'll need](./setup-resources.md)
+- [How to test data source plugins](./test-a-data-source-plugin/index.md)
+<!-- - [How to test panel plugins](./test-a-panel-plugin.md) -->
```

Comment:
The PR that adds this guide is not merged yet. Uncommenting to get the build to pass